app/search.py

```diff
@@ -6,21 +6,47 @@
 """
 
 import webbrowser
-from googlesearch import search
 import sys
+from googlesearch import search
 
 def open_browser(urls):
     for url in urls:
         webbrowser.open_new_tab(url)
 
-def search_my_term(term, count):
-    return [url for url in search(term, stop=count)]
+def search_my_term(term, start, count):
+    return [url for url in search(term, start=start, stop=count)]
+
+def ignore():
+    with open("data/ignore.txt") as f:
+        return f.read().splitlines()
 
+def required(urls, ignore_list):
+    url_req = []
+    for url in urls:
+        flag = 0
+        for domain in ignore_list:
```

Comment:
I missed this code, currently it has two for loops, one better way to do it is

Extract the domain name from URL 
and check `domain in ignore_list`

Can you try that?